users/views.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework.views import APIView
+import rest_framework.permissions as p
+from .serializers import UserSerializer
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from . models import User
+from rest_framework_simplejwt.tokens import RefreshToken
+from django.contrib.auth.signals import user_logged_in
+from rest_framework.generics import RetrieveUpdateAPIView
+from rest_framework.permissions import IsAuthenticated
+
+def get_tokens_for_user(user):
+    refresh = RefreshToken.for_user(user)
+
+    return {
+        'refresh': str(refresh),
+        'access': str(refresh.access_token),
+    }
+
+
+class CreateUserAPIView(APIView):
+    # Allow any user (authenticated or not) to access this url
+    permission_classes = (p.AllowAny,)
+
+    def post(self, request):
+        user = request.data
+        serializer = UserSerializer(data=user)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+@api_view(['POST'])
```

Comment:
Why are you using a different approach here?

Reply:
Basically because I copy-pasted from different sources (sorry) :/
Which approach is more preferable?

Reply:
It is better to stick to one style, of course. BUT if this is training for an alternative view creation, then ok.